src/app/pages/completed-todos/completed-todo.tsx

```diff
@@ -0,0 +1,19 @@
+import Card from "../../../components/ui/card/card";
+import { useTodoStore } from "../../../store/todo/todo-store";
+import TodoSection from "../../../components/ui/todo-section/todo-section";
+import NoTodos from "../../../components/no-todos/no-todos";
+
+export default function CompletedTodo() {
+  const todos = useTodoStore((state) => state.todos);
+  const hasCompletedTodos = todos.filter((todo) => todo.completed === true);
```

Comment:
Since there are only two states, I would store the tasks directly in two different arrays, instead of filtering them in each render of the CompletedTodo and Todo component.